tests/Unit/Subscriptions/Iterators/SyncIteratorTest.php

```diff
@@ -47,10 +47,10 @@ public function testCanPassExceptionToHandler(): void
 
         $this->iterator->process(
```

Comment:
That test seems to cover a property that iterators should generally fulfill.

Reply:
Not exactly sure what you mean with this, this is an pre-existing test, what would you change here?

Reply:
I abstracted the common tests that check if iterators are well behaved.